src/scss/layout/_navigation.scss

```diff
@@ -0,0 +1,69 @@
+.nav {
+  &__button-menu {
+    background: transparent;
+    border: 0;
+    border-radius: 39px;
+    bottom: 20px;
+    box-shadow: 0 0 8px 0 rgba(0, 0, 0, 0.11);
+    outline: 0;
+    position: fixed;
+    right: 20px;
+    width: 60px;
+    z-index: 999;
```

Comment:
This z-index value is not necessary as the elements stack in the order they are defined in the HTML